src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -1654,8 +1654,7 @@ export class ResultsViewPageStore {
         invoke: ()=>{
             return internalClient.fetchCosmicCountsUsingPOST({
                 keywords: _.uniq(this.mutations.result!.filter((m:Mutation)=>{
-                    const simplifiedMutationType = getSimplifiedMutationType(m.mutationType);
-                    return (simplifiedMutationType === "missense" || simplifiedMutationType === "inframe") && !!m.keyword;
+                    return !!m.keyword;
```

Comment:
would love a comment here explaining why we want to filter for items with a keyword.